gf-guide/upstream.md

```diff
@@ -98,7 +98,11 @@ Each file or dir has the following purpose:
 
 -   **[documentation](https://github.com/googlefonts/Unified-Font-Repository/tree/main/documentation)**
     <br>
-    A directory that contains expanded information about the Family. You can eventually store in it your PDF specimen, screenshots or process definition, the pictures you use for the README.md, and [**promotional assets**](marketing.md) for Twitter.
+    A directory that contains expanded information about the Family. You can eventually store in it your PDF specimen, screenshots or process definition, the pictures you use for the README.md. This dir my also include:
+    -  An `article` subdirectory with the images to be included in the [About](./article.md) section.
+    -  An `social-assets` subdirectory with the promotional assets for the social media platforms.
+    -  Please read more about this images in the [Promoting your font](promotion.md) section of this Guide.
```

Comment:
"Please read more about this images" -> "Please read more about these images".

Reply:
Solved